widgets/tab/options.go

```diff
@@ -0,0 +1,113 @@
+// Package tab provides configuration options for the tabbed interface.
+package tab
+
+import "github.com/mum4k/termdash/cell"
+
+// Option represents a configuration option for the Tab.
+type Option interface {
+	// set applies the option to the provided Options struct.
+	set(*Options)
+}
+
+// Options holds the configuration for the Tab.
+type Options struct {
```

Comment:
Similar to treeview and the other tabs - can we make all these fields private (starting with lowercase letter)? Or do they need to be part of the API? Looks like users can use the functions below to set them.